docs/wiki/auto-deploy.md

```diff
@@ -22,25 +22,16 @@ The workflow builds the exact same container as in local development, so the out
 
 The development server which comes with `mkdocs` isn't suited, nor suitable, for a production environment. For this, we build a custom container based off [NGINX](https://hub.docker.com/_/nginx/), which is far better suited, and allows for more control over the server.
 
-### 4. Publish Container
+### 4. Container Update and Prune
 
-Once the production container is built, it's pushed to GitHub's [container registry](https://github.com/selfhostedshow/infrastructure/packages).
-
-### 5. Server Pull
-
-Every 30 seconds, the server polls GitHub's container registry for new containers, using [watchtower](https://containrrr.github.io/watchtower/). When a new container is pulled, it's
-
-### 6. Restart
-
-Once the new container is pulled, watchtower stops the running container, and replaces it with the new one. This is done automatically as quickly as possible, to minimise potential downtime during the switchover.
+Once the build of the production container is complete it is started on the server using `docker-compose`. Within the same step the old image is pruned. This is done automatically as quickly as possible, to minimise potential downtime during the switchover.
```

Comment:
This all looks great, although I think it's probably worth documenting that the build happens on the production server, rather than on CI and pushed via a registry.

Reply:
Thank you for the feedback. Leave it with me, I'll re-work